tests/InstrEngineTests/NaglerInstrumentationMethod/NaglerInstrumentationMethod.cpp

```diff
@@ -899,6 +909,11 @@ HRESULT CInstrumentationMethod::InstrumentMethod(_In_ IMethodInfo* pMethodInfo,
         PerformSingleReturnInstrumentation(pMethodInfo, pInstructionGraph);
     }
 
+    if (pMethodEntry->IsAddExceptionHandler())
+    {
+        AddExceptionHandler(pMethodInfo, pInstructionGraph);
+    }
```

Comment:
Should this be applied after the other instrumentation? Otherwise, people who are writing test scripts will have to infer what transformations are going to be performed by the exception handler rather than just looking at the original IL.

Reply:
Yes, you're right it should probably be last. I've just been testing it on it's own for now, so it doesn't really matter.

Reply:
Fixed here: https://github.com/robertpi/CLRInstrumentationEngine/commit/b1e30e9e351fdd8bb1a9fa7d61d8b301a1bec158